src/App.tsx

```diff
@@ -1,73 +1,37 @@
-import React from 'react';
 import './App.scss';
+import React, { useState } from 'react';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { Autocomplete } from './components/Autocomplete';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+
+  const handleSelectedPerson = (person: Person | null) => {
+    setSelectedPerson(person);
+  };
+
+  const handleInputChange = () => {
+    if (selectedPerson) {
+      setSelectedPerson(null);
+    }
+  };
 
   return (
     <div className="container">
       <main className="section is-flex is-flex-direction-column">
         <h1 className="title" data-cy="title">
-          {`${name} (${born} - ${died})`}
+          {selectedPerson
+            ? `${selectedPerson.name} (${selectedPerson.born} - ${selectedPerson.died})`
+            : 'No selected person'}
```

Comment:
The conditional rendering logic here is correct, but you might want to consider using a more explicit check for `selectedPerson` being `null` to improve readability. For example, `selectedPerson !== null ? ... : ...`.